### PR TITLE
Remove EMSCRIPTEN__TODO and TODO

### PR DESCRIPTION
**Pull request by Maijin**
Remove Emscripten and TODO

### CODE REVIEW

**Review by wargio — COMMENTED** [review]
i'm ok for emscripten but dunno for those TODOs

File: librz/util/debruijn.c [hunk 1]
```diff
@@ -7,7 +7,6 @@
 // char* peda_charset =
 //    "A%sB$nC-(D;)Ea0Fb1Gc2Hd3Ie4Jf5Kg6Lh7Mi8Nj9OkPlQmRnSoTpUqVrWsXtYuZvwxyz";
 
-//TODO(crowell): Make charset configurable, to allow banning characters.
```
Comment (XVilka) [inline librz/util/debruijn.c:10]: I think this TODO should stay, it's really useful feature.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/util/debruijn.c (hunk 1 above)
Comment (Maijin) [reply librz/util/debruijn.c:10]: If it's useful then It should be moved to an issue, not lost in-line TODO

Comment (wargio) [reply librz/util/debruijn.c:10]: i agree with Maijin. that's a valid point.

**Review by XVilka — APPROVED** [review]

**Pull request merged**